bundle/manifests/orchestrator-operator.clusterserviceversion.yaml

```diff
@@ -116,12 +116,12 @@ metadata:
     capabilities: Basic Install
     categories: Developer Tools
     console.openshift.io/disable-operand-delete: "true"
-    createdAt: "2024-06-21T02:10:21Z"
+    createdAt: "2024-06-28T15:09:50Z"
     features.operators.openshift.io/cnf: "false"
     features.operators.openshift.io/cni: "false"
     features.operators.openshift.io/csi: "false"
     features.operators.openshift.io/disconnected: "false"
-    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/fips-compliant: "false"
```

Comment:
iirc, this was introduced in a different PR.
https://github.com/parodos-dev/orchestrator-helm-operator/pull/67/files

Reply:
Yes. I had to create this PR because the pipeline in konflux had issues and could not run at that time. Then, when it was fixed, I was unable to rerun the pipelinerun via UI (invalid pipelinerun it said), so the last resort was to raise this NOOP PR with the only purpose to trigger a new pipelinerun.